### PR TITLE
v0.3.0 release

### PR DESCRIPTION
**Pull request by rebelinux**
Pull request for version 0.3.0

### Changed

- Added additional  health-check section support

- Added additional node section items:

  - Added node vol0 information and health-check

- Added additional storage section items:

  - Added failed disk  health-check

  - Added shelf inventory

  - Added cloud tier (Fabric Pool)

  - Added fabriPool object store configuration information

- Added additional network section items:

  - Added IPSpace, Ifgrp, Vlan, Broadcast Domain, Subnet and Routes section support

    - Added per network interface routes information

  - Added network interfaces (Cluster, Management, Intercluster & Data) information and  health-check

- Added network section:

  - Added SVM Status, Storage Volumes, Volumes QoS Policy, FlexGroup Volumes, Flexclone, Flexcache, Volumes Snapshot, Qtree & Quota

    - Added vserver protocol support (Cifs, NFS, FCP, ISCSI & S3)

      - Added protocol  health-check support

- Added replication section:

  - Added cluster peer information

  - Added vserver peer information

  - Added SnapMirror/SnapVault information and  health-check

  - Added ontap mediators information and  health-check

- Added efficiency section:

  - Added aggregate efficiency information and  health-check

  - Added volume efficiency information and  health-check

- Added security section:

  - Added local user information and  health-check

  - Added vserver ssl certificate information

  - Added Key Management Service (KMS) information and  health-check

  - Added aggregate encryption (NAE) information

  - Added volume encryption (NVE)

  - Added snaplock information

- Added system configuration section:

  - Added system image configuration information

  - Added system web service information

  - Added dns configuration information

  - Added snmp configuration information

  - Added configuration backup information and  health-check

  - Added ems configuration information

  - Added ntp and timezone configuration information

- Fix code logic to better detect of un-configured features

- Added fix for powershell v6+ support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,74 +11,128 @@
...
-    <a href="https://raw.githubusercontent.com/AsBuiltReport/AsBuiltReport.NetApp.ONTAP/master/LICENSE" alt="GitHub License">
-        <img src="https://img.shields.io/github/license/AsBuiltReport/AsBuiltReport.NetApp.ONTAP.svg" /></a>
-    <a href="https://github.com/AsBuiltReport/AsBuiltReport.NetApp.ONTAP/graphs/contributors" alt="GitHub Contributors">
-        <img src="https://img.shields.io/github/contributors/AsBuiltReport/AsBuiltReport.NetApp.ONTAP.svg"/></a>
+    <a href="https://github.com/rebelinux/AsBuiltReport.NetApp.ONTAP/graphs/commit-activity" alt="GitHub Last Commit">
+        <img src="https://img.shields.io/github/last-commit/rebelinux/AsBuiltReport.NetApp.ONTAP/master.svg" /></a>
+    <a href="https://raw.githubusercontent.com/rebelinux/AsBuiltReport.NetApp.ONTAP/master/LICENSE" alt="GitHub License">
+        <img src="https://img.shields.io/github/license/rebelinux/AsBuiltReport.NetApp.ONTAP.svg" /></a>
+    <a href="https://github.com/rebelinux/AsBuiltReport.NetApp.ONTAP/graphs/contributors" alt="GitHub Contributors">
+        <img src="https://img.shields.io/github/contributors/rebelinux/AsBuiltReport.NetApp.ONTAP.svg"/></a>
```
Comment (tpcarman) [inline README.md:19]: These links should not be modified and should point to the AsBuiltReport repo. When committed to the `master` branch they will work correctly.

File: README.md [hunk 2]
```diff
@@ -11,74 +11,128 @@
...
 </p>
 <p align="center">
     <a href="https://twitter.com/AsBuiltReport" alt="Twitter">
             <img src="https://img.shields.io/twitter/follow/AsBuiltReport.svg?style=social"/></a>
 </p>
 
-# NetApp ONTAP As Built Report
+# NetApp Ontap Arrays AsBuiltReport
+
+NetApp Ontap AsBuiltReport is a module of the parent "AsBuiltReport" [project](https://github.com/AsBuiltReport/AsBuiltReport). AsBuiltReport is a PowerShell module which generates As-Built documentation for many common datacentre infrastructure systems. Reports can be generated in Text, XML, HTML and MS Word formats and can be presented with custom styling to align with your company/customer's brand.
```
Comment (tpcarman) [inline README.md:28]: Please remove XML. XML is deprecated.

File: README.md [hunk 3]
```diff
@@ -11,74 +11,128 @@
...
+
+This report is compatible with the following PowerShell versions;
+
+| Windows PowerShell 5.1 | PowerShell Core | PowerShell 7 |
+|:----------------------:|:---------------:|:------------:|
+|   :white_check_mark:   |   :white_check_mark:    |  :white_check_mark:  |
 
 ## :wrench: System Requirements
 
+Each of the following modules will be automatically installed by following the [module installation](https://github.com/rebelinux/AsBuiltReport.NetApp.ONTAP#package-module-installation) procedure.
```
Comment (tpcarman) [inline README.md:76]: Links should be to the AsBuiltReport repo

File: README.md [hunk 4]
```diff
@@ -11,74 +11,128 @@
...
 
 ```powershell
-install-module AsBuiltReport.NetApp.ONTAP
+Install-Module NetApp.ONTAP
+Install-Module AsBuiltReport
 ```
 
-### GitHub
-If you are unable to use the PowerShell Gallery, you can still install the module manually. Ensure you repeat the following steps for the [system requirements](https://github.com/AsBuiltReport/AsBuiltReport.NetApp.ONTAP#wrench-system-requirements) also.
+Ensure you repeat the following steps for the [system requirements](https://github.com/rebelinux/AsBuiltReport.NetApp.ONTAP#wrench-system-requirements) also.
```
Comment (tpcarman) [inline README.md:95]: Fix link

File: README.md [hunk 5]
```diff
@@ -11,74 +11,128 @@
...
 
-1. Download the code package / [latest release](https://github.com/AsBuiltReport/AsBuiltReport.NetApp.ONTAP/releases/latest) zip from GitHub
+1. Download the code package / [latest release](https://github.com/rebelinux/AsBuiltReport.NetApp.ONTAP/releases/latest) zip from GitHub
```
Comment (tpcarman) [inline README.md:97]: Fix link

File: README.md [hunk 6]
```diff
@@ -11,74 +11,128 @@
...
 2. Extract the zip file
-3. Copy the folder `AsBuiltReport.NetApp.ONTAP` to a path that is set in `$env:PSModulePath`.
+3. Copy the folder `AsBuiltReport..NetApp.ONTAP` to a path that is set in `$env:PSModulePath`.
```
Comment (tpcarman) [inline README.md:99]: Remove extra period `.`

File: README.md [hunk 7]
```diff
@@ -11,74 +11,128 @@
...
 
-The NetApp ONTAP As Built Report utilises a JSON file to allow configuration of report information, options, detail and healthchecks. 
+The NetApp Ontap Array As Built Report utilises a JSON file to allow configuration of report information, options, detail and healthchecks.
+
+A NetApp Ontap Array report configuration file can be generated by executing the following command;
 
-A NetApp ONTAP report configuration file can be generated by executing the following command;
 ```powershell
-New-AsBuiltReportConfig -Report AsBuiltReport.NetApp.ONTAP -FolderPath <User specified folder> -Filename <Optional> 
+New-AsBuiltReportConfig -Report NetApp.ONTAP -Path <User specified folder> -Name <Optional>
```
Comment (tpcarman) [inline README.md:120]: `Path` should be `FolderPath` and 'Name` should be `FileName`

File: README.md [hunk 8]
```diff
@@ -11,74 +11,128 @@
...
+
+For detailed documentation around the whole project, please refer to the `README.md` file in the parent AsBuiltReport repository (linked to above). This README is specific only to the NetApp Ontap Array repository.
+
+## :books: Sample Reports
+
+### Sample Report - Custom Style 1
+
+Sample NetApp Ontap As Built report HTML file: [Sample NetApp As-Built Report.html](https://raw.githubusercontent.com/rebelinux/AsBuiltReport.NetApp.ONTAP/dev/Samples/Sample%20NetApp%20As-Built%20Report.html)
+
+Sample NetApp Ontap As Built report with health checks, using custom report style.
```
Comment (tpcarman) [inline README.md:38]: ONTAP should be in uppercase in keeping with NetApp branding, and consistent throughout the docs.

File: Src/Private/Get-AbrOntapClusterLicense.ps1 [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+                }
+            }
+            if ($Healthcheck.License.RiskSummary) {
+                $LicenseSummary | Where-Object { $_.'Risk' -like 'low' } | Set-Style -Style Ok -Property 'Risk'
+                $LicenseSummary | Where-Object { $_.'Risk' -like 'medium' -or $_.'Risk' -like 'unknown' } | Set-Style -Style Warning -Property 'Risk'
+                $LicenseSummary | Where-Object { $_.'Risk' -like 'High' } | Set-Style -Style Critical -Property 'Risk'
+            }
+            $TableParams = @{
+                Name = "License Summary - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapClusterLicense.ps1:45]: Add ColumnWidths

File: Src/Private/Get-AbrOntapClusterLicenseUsage.ps1 [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+                    'Status' = $NodeLFs.Status
+                    'Notes' = Switch ($NodeLFs.Notes) {
+                        "-" { 'None' }
+                        default { $NodeLFs.Notes }
+                    }
+                }
+            }
+            $TableParams = @{
+                Name = "License Feature Summary - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapClusterLicenseUsage.ps1:40]: Add ColumnWidths

File: Src/Private/Get-AbrOntapDiskType.ps1 [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+                    'Container' = $DiskContainers.Name
+                    'Disk Count' = $DiskContainers | Select-Object -ExpandProperty Count
+                    }
+                }
+                if ($Healthcheck.Storage.DiskStatus) {
+                    $DiskType | Where-Object { $_.'Container' -like 'broken' } | Set-Style -Style Critical -Property 'Disk Count'
+                }
+            $TableParams = @{
+                Name = "Disk Container Type Summary - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapDiskType.ps1:39]: Add ColumnWidths

File: Src/Private/Get-AbrOntapNetworkBdomain.ps1 [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+                    'Failover Groups' = $Item.FailoverGroups
+                    'Mtu' = $Item.Mtu
+                    'Ports' = $Item.Ports
+                }
+                $BDomainObj += [pscustomobject]$inobj
+            }
+
+            $TableParams = @{
+                Name = "Network Broadcast Domain Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapNetworkBdomain.ps1:42]: Add ColumnWidths

File: Src/Private/Get-AbrOntapNetworkFailoverGroup.ps1 [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+                    'Name' = $Item.FailoverGroup
+                    'Vserver' = $Item.Vserver
+                    'Target' = $Item.Target
+                }
+                $FGObj += [pscustomobject]$inobj
+            }
+
+            $TableParams = @{
+                Name = "Network Failover Group Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapNetworkFailoverGroup.ps1:40]: Add ColumnWidths

File: Src/Private/Get-AbrOntapNetworkPorts.ps1 [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+                    'Admin State' = $Nics.IsAdministrativeUp
+                }
+            }
+            if ($Healthcheck.Network.Port) {
+                $PhysicalNic | Where-Object { $_.'Link Status' -like 'down' -and $_.'Admin State' -like 'True' } | Set-Style -Style Warning -Property 'Link Status'
+            }
+
+            $TableParams = @{
+                Name = "Physical Port Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapNetworkPorts.ps1:45]: Add ColumnWidths

File: Src/Private/Get-AbrOntapNetworkRoute.ps1 [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+                    'Gateway' = $Item.Gateway
+                    'Metric' = $Item.Metric
+                    'Vserver' = $Item.Vserver
+                }
+                $RoutesObj += [pscustomobject]$inobj
+            }
+
+            $TableParams = @{
+                Name = "Network Route Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapNetworkRoute.ps1:41]: Add ColumnWidths

File: Src/Private/Get-AbrOntapNetworkVlans.ps1 [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+                    'ParentInterface' = $Item.ParentInterface
+                    'VlanID' = $Item.VlanID
+                    'Node' = $Item.Node
+                }
+                $VlanObj += [pscustomobject]$inobj
+            }
+
+            $TableParams = @{
+                Name = "Network VLAN Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapNetworkVlans.ps1:41]: Add ColumnWidths

File: Src/Private/Get-AbrOntapRequiredModule.ps1 [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+    if ($null -eq $OntapModuleVersion) {
+        Write-Warning -Message "Netapp PSTK $OntapRequiredVersion or higher is required to run the NetApp Ontap As Built Report. Run 'Install-Module -Name Netapp.Ontap -MinimumVersion $OntapRequiredVersion' to install the required modules."
+        break
+    } elseif ($OntapModuleVersion -lt $OntapRequiredVersion) {
+        Write-Warning -Message "Netapp PSTK $OntapRequiredVersion or higher is required to run the NetApp Ontap As Built Report. Run 'Update-Module -Name Netapp.Ontap -MinimumVersion $OntapRequiredVersion' to update Netapp.Ontapp modules."
+        break
+    } elseif (Import-Module NetApp.ONTAP) {
+        Import-Module NetApp.ONTAP
+    }
+}
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapRequiredModule.ps1:27]: Remove this file and add the NetApp ONTAP module to `RequiredModules` in the module manifest. 

Originally we made a decision to make users responsible for installing 3rd party modules, however I am now reversing that decision and will include required modules in the module manifest going forward.

File: Src/Private/Get-AbrOntapStorageAGGR.ps1 [hunk 18]
```diff
@@ -0,0 +1,59 @@
...
+                }
+            }
+            if ($Healthcheck.Storage.Aggr) {
+                $AggrSpaceSummary | Where-Object { $_.'State' -eq 'failed' } | Set-Style -Style Critical -Property 'State'
+                $AggrSpaceSummary | Where-Object { $_.'State' -eq 'unknown' -or $_.'State' -eq 'offline' } | Set-Style -Style Warning -Property 'State'
+                $AggrSpaceSummary | Where-Object { $_.'Used' -ge 90 } | Set-Style -Style Critical -Property 'Used'
+            }
+            $TableParams = @{
+                Name = "Aggregate Summary - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapStorageAGGR.ps1:48]: Add ColumnWidths

File: Src/Private/Get-AbrOntapVserverCIFSDC.ps1 [hunk 19]
```diff
@@ -0,0 +1,54 @@
...
+                    'Server Type' = $Item.ServerType
+                    'Prefer Type' = $Item.PreferType
+                    'Status' = $Item.Status
+                }
+                $VserverObj += [pscustomobject]$inobj
+            }
+
+            $TableParams = @{
+                Name = "CIFS Connected Domain Controller Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapVserverCIFSDC.ps1:43]: Add ColumnWidths

File: Src/Private/Get-AbrOntapVserverFcpInterface.ps1 [hunk 20]
```diff
@@ -0,0 +1,52 @@
...
+                    'FCP WWPN' = $Item.PortName
+                    'Home Port' = $Item.CurrentPort
+                    'Vserver' = $Item.Vserver
+                }
+                $VserverObj += [pscustomobject]$inobj
+            }
+
+            $TableParams = @{
+                Name = "FCP Interface Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapVserverFcpInterface.ps1:41]: Add ColumnWidths

File: Src/Private/Get-AbrOntapVserverFcpSummary.ps1 [hunk 21]
```diff
@@ -0,0 +1,57 @@
...
+                }
+                $VserverObj += [pscustomobject]$inobj
+            }
+            if ($Healthcheck.Vserver.FCP) {
+                $VserverObj | Where-Object { $_.'Status' -like 'Down' } | Set-Style -Style Warning -Property 'Status'
+            }
+
+            $TableParams = @{
+                Name = "Vserver FCP Service Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapVserverFcpSummary.ps1:46]: Add ColumnWidths

File: Src/Private/Get-AbrOntapVserverIscsiInitiator.ps1 [hunk 22]
```diff
@@ -0,0 +1,51 @@
...
+                    'Initiator Name' = $Item.InitiatorNodeName
+                    'Target Port Group' = $Item.TpGroupName
+                    'Vserver' = $Item.Vserver
+                }
+                $VserverObj += [pscustomobject]$inobj
+            }
+
+            $TableParams = @{
+                Name = "ISCSI Client Initiator Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapVserverIscsiInitiator.ps1:40]: Add ColumnWidths

File: Src/Private/Get-AbrOntapVserverIscsiInterface.ps1 [hunk 23]
```diff
@@ -0,0 +1,59 @@
...
+                }
+                $VserverObj += [pscustomobject]$inobj
+            }
+            if ($Healthcheck.Vserver.Iscsi) {
+                $VserverObj | Where-Object { $_.'Status' -like 'Down' } | Set-Style -Style Warning -Property 'Status'
+            }
+
+            $TableParams = @{
+                Name = "ISCSI Interface Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapVserverIscsiInterface.ps1:48]: Add ColumnWidths

File: Src/Private/Get-AbrOntapVserverIscsiSummary.ps1 [hunk 24]
```diff
@@ -0,0 +1,58 @@
...
+                }
+                $VserverObj += [pscustomobject]$inobj
+            }
+            if ($Healthcheck.Vserver.Iscsi) {
+                $VserverObj | Where-Object { $_.'Status' -like 'Down' } | Set-Style -Style Warning -Property 'Status'
+            }
+
+            $TableParams = @{
+                Name = "Vserver ISCSI Service Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapVserverIscsiSummary.ps1:47]: Add ColumnWidths

File: Src/Private/Get-AbrOntapVserverS3Summary.ps1 [hunk 25]
```diff
@@ -0,0 +1,57 @@
...
+                        'True' { 'UP' }
+                        'False' { 'Down' }
+                    }
+                }
+                $VserverObj += [pscustomobject]$inobj
+            }
+
+            $TableParams = @{
+                Name = "Vserver S3 Service Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapVserverS3Summary.ps1:46]: Add ColumnWidths

File: Src/Private/Get-AbrOntapVserverS3Summary.ps1 [hunk 26]
```diff
@@ -0,0 +1,57 @@
...
+            foreach ($Item in $VserverData) {
+                $inObj = [ordered] @{
+                    'Vserver' = $Item.Name
+                    'HTTP' = $Item.is_http_enabled
+                    'HTTP Port' = $Item.port
+                    'HTTPS' = $Item.is_https_enabled
+                    'HTTPS Port' = $Item.secure_port
+                    'Status' = switch ($Item.enabled) {
+                        'True' { 'UP' }
+                        'False' { 'Down' }
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapVserverS3Summary.ps1:38]: Fix casing

File: Src/Private/Get-AbrOntapVserverVolumesExportPolicy.ps1 [hunk 27]
```diff
@@ -0,0 +1,55 @@
...
+                    'Ro Rule' = $Item.RoRule
+                    'Rw Rule' = $Item.RwRule
+                    'Vserver' = $Item.Vserver
+                }
+                $VserverObj += [pscustomobject]$inobj
+            }
+
+            $TableParams = @{
+                Name = "Vserver Volume Export Policy Information - $($ClusterInfo.ClusterName)"
+                List = $false
```
Comment (tpcarman) [inline Src/Private/Get-AbrOntapVserverVolumesExportPolicy.ps1:44]: Add ColumnWidths

File: Src/Private/get-AbrOntapApi.ps1 [hunk 28]
```diff
@@ -0,0 +1,86 @@
+function Get-AbrOntapApi {
```
Comment (tpcarman) [inline Src/Private/get-AbrOntapApi.ps1:1]: I'd suggest maybe renaming this function and file to `Get-NetAppOntapAPI` in keeping with how I have done the other API wrappers

File: Src/Public/Invoke-AsBuiltReport.NetApp.ONTAP.ps1 [hunk 29]
```diff
@@ -5,37 +5,770 @@ function Invoke-AsBuiltReport.NetApp.ONTAP {
...
-	# Do not remove or add to these parameters
+
+    # Do not remove or add to these parameters
     param (
         [String[]] $Target,
         [PSCredential] $Credential
     )
 
+    # Check if the required version of Modules are installed
+    Get-AbrOntapRequiredModule
```
Comment (tpcarman) [inline Src/Public/Invoke-AsBuiltReport.NetApp.ONTAP.ps1:25]: No longer required if using `RequiredModules` in the module manifest

File: AsBuiltReport.NetApp.ONTAP.json [hunk 30]
```diff
@@ -12,10 +12,69 @@
 
     },
     "InfoLevel": {
-		"_comment_": "Please refer to the AsBuiltReport project contributing guide for information about how to define InfoLevels.",
-        "_comment_": "0 = Disabled, 1 = Enabled / Summary, 2 = Adv Summary, 3 = Detailed, 4 = Adv Detailed, 5 = Comprehensive"
+        "_comment_": "0 = Disabled, 1 = Enabled / Summary, 2 = Adv Summary, 3 = Detailed, 4 = Adv Detailed, 5 = Comprehensive",
```
Comment (tpcarman) [inline AsBuiltReport.NetApp.ONTAP.json:15]: update this to the following for now or until you add more InfoLevels
`"_comment_": "0 = Disabled, 1 = Enabled`

**Review by tpcarman — CHANGES_REQUESTED** [review]
Absolutely outstanding work @rebelinux!!! 🏆 

Well done! I've made a few comments, only very minor issues. 

I find it quite remarkable that you have achieved so much with such little input or guidance by me, given that I also haven't properly updated the contributing guidelines.

Thanks so much for your efforts. 👍 👏

File: Src/Private/Get-AbrOntapVserverS3Summary.ps1 (hunk 26 above)
Comment (rebelinux) [reply Src/Private/Get-AbrOntapVserverS3Summary.ps1:38]: I don't understand this comment

File: AsBuiltReport.NetApp.ONTAP.json (hunk 30 above)
Comment (rebelinux) [reply AsBuiltReport.NetApp.ONTAP.json:15]: Done!

File: Src/Private/Get-AbrOntapVserverS3Summary.ps1 (hunk 26 above)
Comment (tpcarman) [reply Src/Private/Get-AbrOntapVserverS3Summary.ps1:38]: Letter casing is inconsistent. `UP` is in uppercase and `Down` is in mixed case. Change `UP` to `Up`.

File: Src/Private/Get-AbrOntapClusterLicense.ps1 (hunk 9 above)
Comment (rebelinux) [reply Src/Private/Get-AbrOntapClusterLicense.ps1:45]: Fixed!

**Comment by rebelinux** [comment]
Added all requested modifications (hopefully)

**Pull request merged**